src/components/Map/GoogleLoader/GoogleLoader.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useLoadScript } from '@react-google-maps/api';
+import { LoadingSpinner } from '../../../index';
+
+const GoogleLoader = props => {
```

Comment:
Name is bit weird :) it doesn't load Google, it probably loads `google maps sdk`?

Reply:
GoogleMapsAPILoader?